releases/v1.0.8

```diff
@@ -0,0 +1,39 @@
+Highlights
+
+* Option to set default ActivityOptions and per activityType. Can be done
+through the activityStub, WorkflowImplementationOptions, and on a workflow.
+* Added context propagation for local activities
+* Option to set initial time for workflow tests
+* Implemented a Health Check. By default it's on and checks if the server is up
```

Comment:
We should explicitly call out that this change would break users that deploy workers while server is not up.